tycho-integration-test/src/main.rs

```diff
@@ -99,9 +101,22 @@ impl Debug for Cli {
 #[tokio::main]
 async fn main() -> miette::Result<()> {
     dotenv().ok();
-    tracing_subscriber::fmt()
-        .with_env_filter(EnvFilter::from_default_env())
-        .init();
+
+    // Initialize tracing with OpenTelemetry if configured
+    if let Ok(otlp_exporter_endpoint) = std::env::var("OTLP_EXPORTER_ENDPOINT") {
+        let config = ot::TracingConfig { otlp_exporter_endpoint };
+        ot::init_tracing(config)
+            .wrap_err("Failed to initialize OpenTelemetry tracing")?;
+    } else {
+        tracing_subscriber::fmt()
+            .with_env_filter(EnvFilter::from_default_env())
+            .init();
+    }
```

Comment:
Since tracing is independent from telemetry, maybe this block and the ot::init_tracing function should be moved to a "tracing" module.